engine/apps/api/tests/test_alert_receive_channel.py

```diff
@@ -205,9 +205,9 @@ def test_integration_search(
 @pytest.mark.parametrize(
     "role,expected_status",
     [
-        (Role.ADMIN, status.HTTP_200_OK),
-        (Role.EDITOR, status.HTTP_403_FORBIDDEN),
-        (Role.VIEWER, status.HTTP_403_FORBIDDEN),
+        (LegacyAccessControlRole.ADMIN, status.HTTP_200_OK),
+        (LegacyAccessControlRole.EDITOR, status.HTTP_200_OK),
```

Comment:
create, update, and delete changed here to editor because `RBACPermission.Permissions.INTEGRATIONS_WRITE` is granted to editor, not admin. Did we want to change that?